core/templates/dev/head/domain/skill/skill-rights-backend-api.service.spec.ts

```diff
@@ -16,127 +16,101 @@
  * @fileoverview Unit tests for SkillRightsBackendApiService.
  */
 
-// TODO(#7222): Remove the following block of unnnecessary imports once
-// the code corresponding to the spec is upgraded to Angular 8.
-import { UpgradedServices } from 'services/UpgradedServices';
-// ^^^ This block is to be removed.
-require('App.ts');
-require('domain/skill/skill-rights-backend-api.service.ts');
-require('pages/skill-editor-page/skill-editor-page.controller.ts');
-require('services/csrf-token.service.ts');
-
-describe('Skill rights backend API service', function() {
-  var SkillRightsBackendApiService = null;
-  var $rootScope = null;
-  var $scope = null;
-  var $httpBackend = null;
-  var CsrfService = null;
-
-  beforeEach(angular.mock.module('oppia'));
-  beforeEach(angular.mock.module('oppia', function($provide) {
-    var ugs = new UpgradedServices();
-    for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
-      $provide.value(key, value);
-    }
-  }));
-
-  beforeEach(angular.mock.inject(function($injector, $q) {
-    SkillRightsBackendApiService = $injector.get(
-      'SkillRightsBackendApiService');
-    $rootScope = $injector.get('$rootScope');
-    $scope = $rootScope.$new();
-    $httpBackend = $injector.get('$httpBackend');
-    CsrfService = $injector.get('CsrfTokenService');
-
-    spyOn(CsrfService, 'getTokenAsync').and.callFake(function() {
-      var deferred = $q.defer();
-      deferred.resolve('sample-csrf-token');
-      return deferred.promise;
+import { HttpClientTestingModule, HttpTestingController } from
+  '@angular/common/http/testing';
+import { TestBed, fakeAsync, flushMicrotasks, async } from
+  '@angular/core/testing';
+import { SkillRightsBackendApiService } from
```

Comment:
Add a newline above this to separate the Angular and Oppia imports.

Reply:
Fixed